python/ribasim/ribasim/model.py

```diff
@@ -584,20 +585,20 @@ def plot(
 
         return ax
 
-    def to_xugrid(self, add_flow: bool = False, add_allocation: bool = False):
+    def to_xugrid(self, flow: bool = False, allocation: bool = False):
```

Comment:
This change is breaking but I don't think anyone uses it yet. This brings it more in line with the `to_fews` argument names.

Reply:
Why can't `to_fews` take an `add_network` argument? I think `add_x` is much clearer than `x`.